src/component/user/ResetPassword/ResetPassword.test.tsx

```diff
@@ -15,11 +15,9 @@ test('should render password auth', async () => {
     });
 
     render(
-        <TestContext>
-            <MemoryRouter initialEntries={['/new-user?token=invalid']}>
-                <ResetPassword />
-            </MemoryRouter>
-        </TestContext>
+        <MemoryRouter initialEntries={['/new-user?token=invalid']}>
```

Comment:
I think you can use `render(<ResetPassword />, { route: /new-user?token=invalid })`